sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md

```diff
@@ -2987,6 +2988,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -ContentSecurityPolicyConfigurationSynced
+
+When set to `False` a job will be run in up to 24 hours to populate the Content Security Policy configuration with existing third-party apps in the App Catalog.
+
```

Comment:
I would change it with following:
"Forces sync of Content Security Policies for SharePoint Framework components in the tenant application catalog. 

Running `ResyncContentSecurityPolicyConfigurationEntries` will add new entries to the Content Security Policy configuration if missed by looking at the `cdnBasedPath` property under solution's **./config/write-manifests.json** if present.
The sync may take up to 24 hours to complete."

Also, I would add: "in multi geo environments, Content Security Policy entries are different for each geo" (or something on that line)